src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -128,54 +132,102 @@ void deleteLine() {
         assertThat(지하철노선_조회_결과.statusCode()).isEqualTo(HttpStatus.NOT_FOUND.value());
     }
 
-    private ExtractableResponse<Response> 지하철노선_생성(String name, String color) {
-        // when
-        Map<String, Object> params = new HashMap<>();
-        params.put("name", name);
-        params.put("color", color);
-        params.put("upStationId", 강남역_ID);
-        params.put("downStationId", 교대역_ID);
-        params.put("distance", 10);
-
-        return RestAssured.given().log().all()
-                .body(params)
-                .contentType(MediaType.APPLICATION_JSON_VALUE)
-                .when().post("/lines")
-                .then().log().all()
-                .extract();
+    @Test
+    @DisplayName("역사이에 역을 신규로 등록하는 경우 테스트")
+    void newStationBetweenTheStations() {
+        ExtractableResponse<Response> 지하철노선_생성_결과 = 지하철노선_생성("2호선", "bg-green-600", 교대역_ID, 역삼역_ID);
+        assertThat(지하철노선_생성_결과.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        Long 지하철노선_ID = 지하철노선_생성_결과.jsonPath().getLong("id");
+        ExtractableResponse<Response> 지하철노선_구간_등록_결과 = 지하철노선_구간_등록(지하철노선_ID, 교대역_ID, 강남역_ID, 5);
```

Comment:
해당 경우도 통과해야될 것 같은데 실패하네요 😭 
```suggestion
        ExtractableResponse<Response> 지하철노선_구간_등록_결과 = 지하철노선_구간_등록(지하철노선_ID, 강남역_ID, 역삼역_ID, 5);
```

Reply:
```java
    @Test
    @DisplayName("역사이에 역을 신규로 하행으로 등록하는 경우 테스트")
    void newStationBetweenTheStations2() {
        ExtractableResponse<Response> 지하철노선_생성_결과 = 지하철노선_생성("2호선", "bg-green-600", 교대역_ID, 역삼역_ID);
        assertThat(지하철노선_생성_결과.statusCode()).isEqualTo(HttpStatus.CREATED.value());

        Long 지하철노선_ID = 지하철노선_생성_결과.jsonPath().getLong("id");
        ExtractableResponse<Response> 지하철노선_구간_등록_결과 = 지하철노선_구간_등록(지하철노선_ID, 강남역_ID, 역삼역_ID, 5);

        assertAll(
                () -> assertThat(지하철노선_구간_등록_결과.statusCode()).isEqualTo(HttpStatus.CREATED.value()),
                () -> assertThat(지하철노선_구간_등록_결과.jsonPath().getList("stations.name", String.class))
                        .containsExactly("교대역", "강남역", "역삼역")
        );
    }
```

Reply:
하나 추가했습니다 ( \_ \_) 감사합니다